languages/ruby/spec/oso/oso_spec.rb

```diff
@@ -80,14 +80,14 @@ def initialize(id:)
         subject.register_class(Widget)
         subject.load_str <<~POLAR
           allow(actor, "get", _: Http{path: path}) if
-              new PathMapper{template: "/widget/{id}"}.map(path) = {id: id} and
+              new PathMapper("/widget/{id}").map(path) = {id: id} and
               allow(actor, "get", new Widget{id: id});
           allow(_actor, "get", widget: Widget{}) if widget.id = "12";
         POLAR
-        widget12 = Oso::Http.new(path: '/widget/12')
+        widget12 = Oso::Http.new('host', '/widget/12', {})
```

Comment:
What is this doing?

Reply:
See above; because Java has to use this interface, we have to use positional args, so a host and query must be provided.